chxtools/chx_utilities.py

```diff
@@ -1505,14 +1509,15 @@ def get_ID_calibration(gapstart,gapstop,xray_eye1=xray_eye1, gapstep=.2,gapoff=0
     print('using ',xtal,' for ID gap calibration')
     # create file for writing calibration data:
     fn='id_CHX_IVU20_'+str(time.strftime("%m"))+str(time.strftime("%d"))+str(time.strftime("%Y"))+'.dat'
-    fpath='/home/xf11id/Repos/chxtools/chxtools/X-ray_database/'
+    # fpath='/home/xf11id/Repos/chxtools/chxtools/X-ray_database/'
+    dat_file = Path(rs_fn('chxtools', 'X-ray_database')) / Path(fn)
     try:
-      outFile = open(fpath+fn, 'w')
+      outFile = open(str(dat_file), 'w')

```

Comment:
I don't think you need to use `str` here. `open(dat_file, 'w')` seems to work fine, though the usage suggested by `pathlib` is `dat_file.open('w')`.

Reply:
Good catch! I also found `dat_file.write_text(...)` should also work, but we can have more conservative approach.